android/build.gradle

```diff
@@ -27,6 +27,9 @@ buildscript {
     if (project.hasProperty("filament_dist_dir")) {
         filamentPath = file(project.property("filament_dist_dir")).absolutePath
     }
+    if (System.properties['os.name'] != null && System.properties['os.name'].toLowerCase().contains('windows')) {
+        filamentPath = filamentPath.replaceAll("\\\\", "/")
+    }
```

Comment:
```suggestion
    // Our CMake scripts require a forward-slash path for the FILAMENT_DIST_DIR
    // variable, so here we convert the native path to a forward-slash path.
    filamentPath = filamentPath.replaceAll(Path.separator, "/")
```

Reply:
in that case it should be `filamentPath.replace(File.separator, "/")`